src/common/Hero/Hero.tsx

```diff
@@ -0,0 +1,102 @@
+import { Box, Grid, Typography, useTheme, useMediaQuery } from '@mui/material';
+import MemberHero from '@src/assets/images/member-hero.png';
+import Avatar01 from '@src/assets/icons/avatar01.svg?react';
+import Avatar02 from '@src/assets/icons/avatar02.svg?react';
+import Avatar03 from '@src/assets/icons/avatar03.svg?react';
+import Avatar04 from '@src/assets/icons/avatar04.svg?react';
+import Avatar05 from '@src/assets/icons/avatar05.svg?react';
+import Avatar06 from '@src/assets/icons/avatar06.svg?react';
+
+const avatarComponents = [Avatar01, Avatar02, Avatar03, Avatar04, Avatar05, Avatar06];
+
+const RandomIcon = () => {
+  const { breakpoints } = useTheme();
+  const isMediumOrLarger = useMediaQuery(breakpoints.up('md'));
+  const Icon = avatarComponents[Math.floor(Math.random() * avatarComponents.length)];
+  const MobileIcon = () => {
+    return (
+      <Box
+        sx={{
+          width: '72px',
+          height: '72px',
+        }}
+      >
+        <Icon style={{ width: '100%', height: '100%' }} />
+      </Box>
+    );
+  };
+  const DesktopIcon = () => {
+    return (
+      <Box
+        sx={{
+          width: '144px',
+          height: '144px',
+        }}
+      >
+        <Icon style={{ width: '100%', height: '100%' }} />
+      </Box>
+    );
+  };
+  return isMediumOrLarger ? <DesktopIcon /> : <MobileIcon />;
```

Comment:
小地方～不調整也沒關係

已知 DesktopIcon 會回傳元件，所以可以直接回傳變數，可省略元件的寫法

```tsx
return isMediumOrLarger ? DesktopIcon : MobileIcon
```

Reply:
這裡我原本也這麼寫，結果發現好像不行
`Warning: Functions are not valid as a React child. This may happen if you return a Component instead of <Component /> from render. Or maybe you meant to call this function rather than return it.`